### ISSUE TITLE
Support for ALC299

### ISSUE THREAD

**Issue by silent3rror** [insanelydeepak/Patched-AppleHDA-for-Mac-OS-Sierra-10.12#4]
Need support for 0x10ec0299
```
Codec: Realtek Generic
Address: 0
AFG Function Id: 0x1 (unsol 1)
Vendor Id: 0x10ec0299
```

**Comment by insanelydeepak** [comment]
post your codec dump

**Comment by insanelydeepak** [comment]
Hi 
I'm closing issue now , if it still then
feel free to re-open 

Thanks

**Comment by cepxuo** [comment]
Is it possible to re-activate request? Here is my codec dump:
[dump.txt](https://github.com/insanelydeepak/Patched-AppleHDA-for-Mac-OS-Sierra-10.12/files/2097017/dump.txt)
